expyfun/io/tests/test_parse.py

```diff
@@ -48,13 +48,37 @@ def test_parse():
     assert_equal(len(data[0]['misc']), 2)  # includes between-trials stuff
     assert_equal(len(data[1]['misc']), 2)
 
-def test_reconstruct_tracker():
-    """Test reconstruction of TrackerUD and TrackerBinom objects from .tab
-    log files."""
-    from expyfun.io import reconstruct_tracker
+def test_reconstruct():
+    """Test Tracker objects reconstruction"""
+    from expyfun.io import reconstruct_tracker, reconstruct_dealer
```

Comment:
This is yielding me another fun circular import problem. Still pondering how to fix it.